model/labelname.go

```diff
@@ -21,9 +21,9 @@ import (
 )
 
 const (
-	// ExporterLabelPrefix is the label name prefix to prepend if a
+	// ExportedLabelPrefix is the label name prefix to prepend if a
 	// synthetic label is already present in the exported metrics.
```

Comment:
The doc comment should be updated to reflect the priority.

Suggestion:

ExportedLabelPrefix is the prefix to prepend to the label names present in exported metrics if a label of the same name is added by the server.